lms/envs/aws.py

```diff
@@ -676,6 +676,12 @@
         'courseware.student_field_overrides.IndividualStudentOverrideProvider',
     )
 
+##### Self-Paced Course Due Dates #####
+if FEATURES.get('ENABLE_SELF_PACED_COURSES'):
+    FIELD_OVERRIDE_PROVIDERS += (
+        'courseware.self_paced_overrides.SelfPacedDateOverrideProvider',
+    )
```

Comment:
I don't see any reason why not; I was just following the lead of the other overrides above.